modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/test/js/components/RuleList/RuleList.es.js

```diff
@@ -65,11 +64,49 @@ const configDefault = {
 							value: 'value 2',
 						},
 					],
+					operator: 'contains',
+				},
+				{
+					operands: [
+						{
+							type: 'field',
+							value: 'text1',
+						},
+						{
+							type: 'field',
+							value: 'text2',
+						},
+					],
 					operator: 'equals-to',
 				},
 			],
 			['logical-operator']: 'OR',
 		},
+		{
+			actions: [
+				{
+					action: 'show',
+					label: 'label text 2',
+					target: 'text2',
+				},
+			],
+			conditions: [
+				{
+					operands: [
+						{
+							type: 'field',
+							value: 'text1',
+						},
+						{
+							type: 'value',
+							value: 'value 3',
+						},
+					],
+					operator: 'not-equals-to',
+				},
+			],
+			['logical-operator']: 'AND',
```

Comment:
@carolmariaabb, is the logical operator mandatory if there is just one condition?

Reply:
@natocesarrego If you consider the RuleList.es.js code, it is mandatory because of [RuleList.es.js#L60-L142](https://github.com/carolmariaabb/liferay-portal/blob/7a8b847a3f3e8bc649d94cb80248623ea2bde5a2/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleList/RuleList.es.js#L60-L142).